opentelemetry-api/README.md

```diff
@@ -1,28 +0,0 @@
-![OpenTelemetry — An observability framework for cloud-native software.][splash]
```

Comment:
it may be a good idea to to keep this file, with a note saying "its no longer maintained and users should use opentelemetry crate instead". Or maybe that can be done in crates.io description manually.

Reply:
+1 we will need to figure out a way to prompt users to `opentelemetry` crate when they end up on the `opentelemetry_api` crate

Reply:
Could submit an informational RustSec security advisory setting the status for opentelemetry_api to unmaintained.

Reply:
I've submitted https://github.com/rustsec/advisory-db/issues/1994 and a PR